src/components/chat/comment-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import { BaseComponent } from "../base";
+import DocumentCommentIcon from "../../assets/document-id.svg";
+import SendIcon from "../../assets/send-icon.svg";
+import "./comment-card.scss";
+import "../themes.scss";
+
+interface IProps {
+}
+
+interface IState {
+  commentTextAreaHeight: number | string;
+  commentAdded: boolean;
+  commentText: string;
+}
+
+@inject("stores")
+@observer
+export class CommentCard extends BaseComponent<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+    this.handleCommentTextAreaChange = this.handleCommentTextAreaChange.bind(this);
+    this.handleCancelPost = this.handleCancelPost.bind(this);
+    this.handleSendPost = this.handleSendPost.bind(this);
+
+    this.state = {
+      commentTextAreaHeight: 35,
+      commentAdded: false,
+      commentText: "",
+    };
```

Comment:
You can define this outside of the constructor:
```typescript
  state = {
    commentTextAreaHeight: 35,
    commentAdded: false,
    commentText: "",
  };
```
and then you can eliminate the constructor altogether.